src/algebra/category/Module/monoidal.lean

```diff
@@ -320,4 +323,21 @@ lemma monoidal_closed_uncurry {M N P : Module.{u} R}
   monoidal_closed.uncurry f (x ⊗ₜ[R] y) = (@coe_fn _ _ linear_map.has_coe_to_fun (f y)) x :=
 by { simp only [monoidal_closed.uncurry, ihom.adjunction, is_left_adjoint.adj], simp, }
 
+lemma ihom_ev_app (M N : Module.{u} R) :
+  (ihom.ev M).app N = tensor_product.uncurry _ _ _ _ linear_map.id.flip :=
+begin
+  refine tensor_product.ext (linear_map.ext (λ x, linear_map.ext (λ y, _))),
+  simp only [linear_map.compr₂_apply, tensor_product.mk_apply, tensor_product.uncurry_apply,
+    linear_map.flip_apply, linear_map.id_coe, id.def],
```

Comment:
I think these are all refl lemmas, so you might be able to drop this `simp` call entirely.

Reply:
Thank you!